wave_vision/tests/viz_tests/tracker_viz_tests.cpp

```diff
@@ -45,10 +47,48 @@ TEST(TrackerTests, OfflineTrackerTest) {
          img_seq_it++) {
         cv::imshow("Feature Tracks", *img_seq_it);
         cv::waitKey(0);
+        cv::destroyAllWindows();
     }
 }
 
-TEST(TrackerTests, KITTIOfflineTrackerTest) {
+TEST(TrackerTests, ORBOfflineTrackerTest) {
+    std::vector<cv::Mat> image_sequence;
+    std::vector<cv::Mat> drawn_images;
+
+    std::vector<std::vector<FeatureTrack>> feature_tracks;
+
+    ORBDetector detector;
+    ORBDescriptor descriptor;
+    BruteForceMatcher matcher;
+
+    Tracker<ORBDetector, ORBDescriptor, BruteForceMatcher> tracker(
+      detector, descriptor, matcher);
+
+    image_sequence = readImageSequence(FIRST_IMG_PATH);
+
+    feature_tracks = tracker.offlineTracker(image_sequence);
+    ASSERT_NE((int) feature_tracks.size(), 0);
```

Comment:
I like to use `u` suffix to avoid casts in these checks. E.g.  `ASSERT_NE(0u, feature_tracks.size())`;

Here could also do `ASSERT_FALSE(feature_tracks.empty())`